hydrocarbon_problem/env/env.py

```diff
@@ -7,7 +7,7 @@
 from hydrocarbon_problem.api.api_base import BaseAspenDistillationAPI
 from hydrocarbon_problem.api.types_ import StreamSpecification, PerCompoundProperty, \
     ColumnInputSpecification, ColumnOutputSpecification
-from hydrocarbon_problem.env.types_ import Stream, Column, Observation, Done
+from hydrocarbon_problem.env.types import Stream, Column, Observation, Done
```

Comment:
This change should be reverted back to how the file was before:
`from hydrocarbon_problem.env.types_ import Stream, Column, Observation, Done`